### PR TITLE
Cortx 32767

### PR DESCRIPTION
**Pull request by VikasKumarC**
## Problem Statement

-   Dev Setup

## Design

-   For Bug, Describe the fix here.
-   For new test addition post the link for design

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [ ]  Attach test execution logs

## Review 

  Checklist for Author

-   [ ]  JIRA number/GitHub Issue added to PR
-   [ ]  PR is self reviewed
-   [ ]  Jira and state/status is updated and JIRA is updated with PR link
-   [ ]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

File: src/commons/utils/jira_utils.py [hunk 1]
```diff
@@ -34,9 +34,7 @@
 from requests.exceptions import RequestException
 from requests.packages.urllib3.util.retry import Retry # pylint: disable=import-error
 
-from src.commons.constants import ROOT
-
-LOGGER = logging.getLogger(ROOT)
+LOGGER = logging.getLogger(__name__)
```
Comment (DhobaleVishal) [inline src/commons/utils/jira_utils.py:37]: Revert this change as we are logging common & utils related logs in CORIO log file.

File: src/commons/utils/jira_utils.py [hunk 2]
```diff
@@ -189,13 +188,6 @@ def get_all_tests_details_from_tp(self, tp_id: str, reset_status: bool = False)
                                     f" TE '{texe['key']}'")
                 tests_dict[test['key']] = test
                 tests_dict[test['key']]["te"] = texe
-                if reset_status:
```
Comment (DhobaleVishal) [inline src/commons/utils/jira_utils.py:192]: Why this code is removed? This is optional and parameterized so that user can decide to reset jira status or not.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]
@VikasKumarC Please check inline commnts and also update PR title.

**Comment by VikasKumarC** [comment]
Not required
